GOVERNANCE.md

```diff
@@ -0,0 +1,98 @@
+# Project Contribution Guidelines
```

Comment:
Hmm... shouldn't this be "Governance" or something similar?

Contributing Guidelines is in `CONTRIBUTING.md`...

Reply:
it was like that before, see comment by @Afauvre and my reply above.

Reply:
Changed it back to "Governance Model".

Reply:
Aha, I didn't read that before.

I think "Governance Model" and `GOVERNANCE.md` is fine since this governs how the Oasis Core project is run.

Of course, there is the _governance_ of the Oasis Network but that should be described in a different place, probably a document in https://github.com/oasisprotocol/docs.